plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/src/com/google/cloud/tools/eclipse/appengine/standard/java8/messages.properties

```diff
@@ -0,0 +1,2 @@
+reconfiguringProject=Reconfiguring {0} 
+reconfiguringToJava8=Reconfiguring projects to use App Engine Java 8 runtime
```

Comment:
project or projects?

Reply:
Plural: handy for those with multiple projects.